functions_advanced/task3.js

```diff
@@ -0,0 +1,40 @@
+function divide(numerator, denominator) {
+  if (denominator === 0) {
+    throw new Error("Denominator can't be 0");
+  } else if (isNaN(numerator) || isNaN(denominator)) {
+    throw new Error("All arguments should be numbers");
+  }
```

Comment:
better use two `if` blocks instead. It would be more readable. As recomendation

Reply:
Changed it too, indeed it's more readable now.